src/sidebar/helpers/test/mentions-test.js

```diff
@@ -40,9 +40,21 @@ look at ${mentionTag('foo', 'example.com')} comment`,
   },
   // Multiple mentions
   {
-    text: 'Hey @jane look at this quote from @rob',
+    text: 'Hey @jane, look at this quote from @rob',
```

Comment:
When I first implemented this test I added a comma here and the test failed. That's what made me find this bug.